### DISCUSSION TITLE
[Help]: Snap to, with inertia/momentum

### DISCUSSION THREAD

**Discussion by glomotion** [davidjerleke/embla-carousel#850] [Help] [answered]
### Which variants of Embla Carousel are relevant to this feature request?

- [X] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### Feature description

Just wondering if there might be any interest in developing a new feature for this carousel, which essentially combines the best bits (IMO) of both display modes (drag free & snap to).

The idea being that you can flick and scroll through multiple slides, but at the end of the drag event, the inertia resulting from the drag would end landing the user on the closest slide. 

### Additional Context

So this sort of thing does exist in other libraries. For example, in keen-slider, its a mode called "free-snap": https://codesandbox.io/p/sandbox/github/rcbyr/keen-slider-sandboxes/tree/v6/basic/free-snap-mode/react-typescript?file=%2Fsrc%2FApp.tsx

I'd just use keen-slider, however embla carousel's api etc looks very nice, plus I'm a little put off by the fact that keen-slider seems to be completely untested. 😅 

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues
- [X] I agree to follow this project's Contributing Guidelines for feature requests

**Comment by meirroth** [comment, marked as answer]
@glomotion I believe [skipSnaps](https://www.embla-carousel.com/api/options/#skipsnaps) is what you're looking for! I always use it 😃 [See example](https://stackblitz.com/edit/github-pf8j7x?file=app.vue).

**Reply by glomotion** [reply]
AMAZING! thank you for pointing this out. This should be exactly what i need! :D

**Reply by meirroth** [reply]
@glomotion Hi, pls mark comment as answer if it solved your problem. Thanks!

**Reply by glomotion** [reply]
@meirroth how do i do that?

**Reply by meirroth** [reply]
@glomotion Never mind I don't think you can in this type of thread 😅

**Reply by davidjerleke** [reply]
@glomotion was @meirroth‘a answer what you were looking for?
